frontend/src/components/SearchBarFilter.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable array-callback-return */
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
+import React, { useState } from 'react'
+import { Tag } from './Tag'
+import './searchbar.css'
+import { type Friend } from './PlanForm'
+
+interface Props {
+  items: Friend[]
+  handleSelectChange: (friend: Friend) => void
+  handleRemoveTag?: (friend: Friend[]) => void
+  numbOfSelect?: number
```

Comment:
nit: when shortening `number` it's more common to use `num` 😂

Reply:
😂